pkg/collector/corechecks/snmp/report_metrics_test.go

```diff
@@ -0,0 +1,322 @@
+package snmp
+
+import (
+	"bufio"
+	"bytes"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/mocksender"
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/cihub/seelog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"strings"
+	"testing"
+)
+
+func TestSendMetric(t *testing.T) {
+	tests := []struct {
```

Comment:
Maybe we could parse some JSON data test files for better readability...

Reply:
Hum. I'm not sure to understand. You mean the test cases here ? https://github.com/DataDog/datadog-agent/blob/f72c4c551d515e2873937ff00087d8570b5115d6/pkg/collector/corechecks/snmp/report_metrics_test.go#L17-L51 that seems readable to me :)